### DISCUSSION TITLE
Jetbrains licenses

### DISCUSSION THREAD

**Discussion by jekkos** [opensourcepos/opensourcepos#3662] [Team Posts]
Guys, just as last year I asked for a number of jetbrains products licenses, which are free for opensource, in this case also phpstorm. Let me know if interested so I can give you for working on ospos from time to time.

**Comment by odiea** [comment]
If no one else requests one I would like to try it out..

**Comment by WebShells** [comment]
Me too... I'd like to discover and i guess phpstorm helps and cuts the work somehow..

**Comment by jekkos** [comment]
https://account.jetbrains.com/a/co39b36b

**Comment by jekkos** [comment]
You should be able to link the license through this links (you'll need a jetbrains account thoug)

**Comment by jekkos** [comment]
Ok it seems they currently only gave us two licenses, so I asked for two more. Just in case.

**Comment by odiea** [comment]
If there are any left after the real programmers get what they need then I would like 1 just to play around with it. So no biggie for me. Thanks for the offer.

**Comment by WebShells** [comment]
@jekkos  thanks, got it... Activated till September 2019

**Comment by jekkos** [comment]
well I think there should be 4 licenses in total, so go ahead and try the link @odiea

**Comment by odiea** [comment]
Got 1 thanks. A new toy to play with.
